packages/contracts/contracts/libs/v0.8.x/Bytes32Strings.sol

```diff
@@ -63,4 +63,23 @@ library Bytes32Strings {
         }
         return _occurrences == targetQty;
     }
+
+    /**
+     * @notice Converts string to bytes32.
+     * https://ethereum.stackexchange.com/a/9152
+     * @param source string to be converted.
+     * @return result bytes32 representation of string.
```

Comment:
I think worth adding a note that strings whose representations are >32 bytes will be truncated. Additionally, I'm not entirely sure what would happen with a string with a multi-byte character that spans across the end of the 32'nd byte, might throw in that case?

Reply:
I don't think it would throw, I'd imagine it would still just return truncated bytes that would be parsed to a garbled string

Reply:
might be good to test this out and determine if solidity throws an "invalid continuation byte" or something. just want to make sure we document any revert behavior to prevent any denial of service bugs if this were to be used on any future contracts.

Reply:
I do think most likely would be a garbled string though 🤣